CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+Contributing
+============
+
+A guide to contributing to Node Test Helpers.
+
+## Preparing to Fork
+The first thing you'll want to do is making sure your IDE/editor of choice has a few things:
+1. [ESLint] is installed and you can utilize the [ESLint] configuration created in this repo. If you are unfamiliar with [ESLint], check out the site and see how great it is at keeping code consistent and check for some bad lines of code.
+2. [EditorConfig] is installed. Again, if unfamiliar with this tool, please take a few moments to familiarize yourself with it as this is also a handy tool.
+
+## Forking the Repository
+1. Fork the repository from https://github/tests-always-included/jasmine-node-helpers. Please do not fork from a fork.
```

Comment:
Add a newline between lists and headings.

Really, add a newline to separate lists from anything that isn't part of the list.